website/docs/components/pagination/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,51 @@
+## Anatomy
+
+<!-- image then table -->
+### Pagination
+
+![Anatomy of the pagination](/assets/components/pagination/pagination-anatomy.png)
+
+| Element          | Usage                                           |
+|------------------|-------------------------------------------------|
+| Info             | Optional                                        |
+| Nav              | Required                                        |
```

Comment:
We don't often use the shorthand of words. Wondering if this should be "Navigation" instead.

Reply:
The underlying HTML element the component is wrapped with is the [HTML `nav` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).

If we really don't want to use "Nav" though and prefer "Navigation" then would we also want to call the informational component "Information" vs. "Info"?

For me, I tend to prefer the shorter versions in this case, but we can go with whatever the team thinks makes sense.

Reply:
Ah, this is good context. I did not realize it was using the `nav` element. Feel free to ignore my original comment 🙂